### PR TITLE
Add 1990-2000 to the set of EIA 861 years

### PR DESCRIPTION
**Pull request by karldw**
Changes: 

- <s>Pick the "reformatted" version of the EIA file when available (1990-2011)</s> Edit: Pick the "original" version for all years.
- Fix some small typos in the README
- Update the `eia861.html` test data page

Notes: 

1. <s>Please don't merge yet, until I can change the EIA 861 file mapping in pudl.</s> Edit: we've decided not to change the existing files or mapping in pudl, so feel free to merge!
1. <s>I selected the "reformatted" files for the whole 1990-2011 period. If you think it'd be better to use the original files for the whole period, or to use the original files for 2001-2011 (the period that's already included) and the reformatted files for 1990-2000, let me know!</s> Edit: Pick the "original" version for all years.

cc @aesharpe 

Addresses part of https://github.com/catalyst-cooperative/pudl/issues/923

### CODE REVIEW

**Comment by karldw** [comment]
FYI, tests are passing here, but the check fails because of `SLACK_WEBHOOK_URL`.

File: pudl_scrapers/spiders/eia861.py [hunk 1]
```diff
@@ -57,43 +57,57 @@ def all_forms(self, response):
...
+        # Since April or May 2020, the EIA website has provided "original" and
+        # "reformatted" versions of the data for 1990-2011. Select the
+        # reformatted data by taking the last element of the selected list.
+        # For 2012 and later, the list has one element. For 2011 and earlier,
+        # "original" is first and "reformatted" is second.
+        # https://web.archive.org/web/20200509062741/https://www.eia.gov/electricity/data/eia861/
+        link_list = response.xpath(path).getall()
+
+        try:
+            link = link_list[-1]
```
Comment (zaneselvans) [inline pudl_scrapers/spiders/eia861.py]: Oh, that's a nice simple way to do this. Great!

**Review by zaneselvans — APPROVED** [review]
This looks good to me. Nice simple way to grab the reformatted files when they're available, which will hopefully simplify the mapping metadata for the data source overall.  I wish there was a way to allow failure on a particular action within a workflow without the whole thing failing. But it [seems like there isn't right now](https://github.com/actions/toolkit/issues/399).

**Comment by karldw** [comment]
> FYI, tests are passing here, but the check fails because of SLACK_WEBHOOK_URL.

Again.

**Comment by zaneselvans** [comment]
@karldw Is this still a draft? Or shall we merge it and try creating a new archive?  I guess we'll need to update things in the `pudl-zenodo-storage` repo as well.

**Comment by karldw** [comment]
Feel free to merge!

**Pull request merged**